README.md

```diff
@@ -117,12 +117,12 @@ Follow the given instructions for Login into the app.
 
 ## Environment Variables
 
-Make sure you create .env using .env.template and update the values of corresponding environment variables:
+Use `.env.template` file to make a new `.env` file and add the following details:
 
 ```
 SECRET_KEY=<your-secret-key>
 EMAIL_USER=<email-address>
-EMAIL_PASSWORD=<your-password>
+EMAIL_PASSWORD=<your-api-key>
```

Comment:
Remove these lines. I think it is not required as env-variables are defined.
@Aaishpra Needs your suggestion here.

Reply:
Sure @codesankalp I will remove it.

Reply:
@codesankalp please see my latest two commits I think you have seen only the first one in the second commit `Updated Readme` I have removed the EMAIL_PASSWORD variable sorry for the confusion.

Reply:
Let them be there, might be useful beginners